python/py-awscli/Portfile

```diff
@@ -30,15 +29,12 @@ if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools
     depends_lib-append \
-                        port:py${python.version}-asn1 \
-                        port:py${python.version}-bcdoc \
                         port:py${python.version}-botocore \
                         port:py${python.version}-colorama \
-                        port:py${python.version}-dateutil \
                         port:py${python.version}-docutils \
-                        port:py${python.version}-jmespath \
                         port:py${python.version}-rsa \
-                        port:py${python.version}-six
```

Comment:
A port **must** declare dependencies on all other software it uses, even if another dependency includes them also. Transitive dependencies are very fragile (and misleading besides).

I did a cursory textual search for a couple of the removed dependencies. Looks like some of the "customizations" scripts use them, but I don't know what those scripts are for or whether they're installed. We might be able to ignore them entirely.